src/zabapgit_unit_test.prog.abap

```diff
@@ -2754,4 +2759,100 @@ CLASS ltcl_interface_deserialization IMPLEMENTATION.
 
     then_docu_should_be_created( lt_lines ).
   ENDMETHOD.
+ENDCLASS.
+
+CLASS ltcl_class_changed DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT
```

Comment:
perhaps split the unit tests relevant for CLAS + INTF into a separate include to avoid merging conflicts eg. zabapgit_unit_test_clas.prog.abap